onnxruntime-sys/build.rs

```diff
@@ -64,10 +64,19 @@ fn main() {
     let libort_install_dir = prepare_libort_dir();
 
     #[cfg(not(feature = "directml"))]
```

Comment:
directmlの場合はcfgで制御、androidの場合はmatchで制御、と不揃いになっているので、どちらかに統一するとわかりやすそうだなとちょっと思いました。
が、このPRとしてはこれで問題ないので、ちょっとFIXMEコメントだけお願いできると！
```suggestion
    // FIXME: directmlとandroidで処理の表現が違うので統一する
    #[cfg(not(feature = "directml"))]
```